src/components/collapsibleContainer/CollapsibleContainer.tsx

```diff
@@ -71,7 +71,7 @@ export class CollapsibleContainer extends React.Component<ICollapsibleContainerP
     }
 
     const tooltipProps = { placement: 'right', ...this.props.informationTooltip };
-    const svgProps = { svgName: 'help', svgClass: this.getSvgClass() };
+    const svgProps = { svgName: this.props.informationUrl ? 'help' : 'info', svgClass: this.getSvgClass() };
```

Comment:
It was `help-no-fill-18` before, will this change anything?

Reply:
you can have a look by searching collapsiblecontainer here https://coveo.github.io/react-vapor/info-instead-of-help/ , let me know what you think  

looks fine i wonder if we should add something to white-fill the inside of the icon

Reply:
Personally, I prefer when its not white filled. IMHO transparency looks better

Reply:
I understand the difference now. The background of the icon is wanted. That was a request from the UX team, so we shouldn't even use `help-no-fill`. The background color is also the right one.

Reply:
cool thanks guys